.github/workflows/contract-test.yml

```diff
@@ -66,3 +66,4 @@ jobs:
       env:
         PRIVATE_KEY: ${{ secrets.GODWOKEN_TEST_PRIVATE_KEY }}
         PRIVATE_KEY2: ${{ secrets.GODWOKEN_TEST_PRIVATE_KEY2 }}
+        PRIVATE_KEY3: ${{ secrets.GW_TESTNET_V1_TEST_PK }}
```

Comment:
Should we add the new secret in job `check-secrets`?

Reply:
1b3727985540ad7905fa7d042f66d4b47fef07ca